src/mudlet.cpp

```diff
@@ -2952,8 +2952,7 @@ void mudlet::slot_connection_dlg_finished(const QString& profile, bool connect)
 
     pHost->mBlockStopWatchCreation = false;
     pHost->getScriptUnit()->compileAll();
-    pHost->mIsProfileLoadingSequence = false;
```

Comment:
`(bool) Host::mIsProfileLoadingSequence` is currently (only) used to **force** the (test?) compilation of the Lua script for Push-down type buttons when Toolbars or Menus that contain them are (re)generated - is the relocation of this change of that flag so it can be used for something else as well going to compromise the fix that it was originally done for?

Reply:
Relocation is actually fix... 
https://github.com/Mudlet/Mudlet/pull/4508

Reply:
Yeah - so you can use the same flag to prevent a save action from occurring - but does moving the place where you change the state of that flag mess with what it was **originally** doing (forcing the scripts associated with push-down buttons to be executed specifically during toolbar/menu regeneration during loading)? Does anyone here remember why Heiko did 3207ccc97c66c9e84369cdd17a9710ef5867c81e and 3e127f554bb23f8e86fbe64c560bccfc7bd096d0 (even the first of those was before my time)?

Reply:
This kind of extends `mIsProfileLoadingSequence` to modules as well, so I'd say we should be fine here.